msteams-platform/m365-apps/extend-m365-teams-message-extension.md

```diff
@@ -30,7 +30,11 @@ To complete this tutorial, you'll need:
 * (Optional) Microsoft Visual Studio Code with the Teams Toolkit extension.
 
 > [!div class="nextstepaction"]
-> [Publish Teams apps extended for Microsoft 365](publish.md)
+> [Install prerequisites](prerequisites.md)
+
+## Link unfurling
+
+If your search-based message extension supports [link unfurling](../messaging-extensions/how-to/link-unfurling.md) in Teams, completing the steps of this tutorial will also enable link unfurling in Outlook on the web and Windows desktop environments. The [Code samples](#code-sample) section below provides a simple link unfurling app for testing.
```

Comment:
Please see if "will" can be removed. As per standards we need to avoid usage of future tense wherever possible.